### PR TITLE
Added iOS info

### PR DESCRIPTION
**Pull request by sm0rux**
Not sure, but I think the iOS app handles "WireGuard" and "Optional local network access" as the Android app does.

Describe **what** this PR changes. **Why** this is wanted. And, if needed, **how** it does it.

Git checklist:

* [ ] Describe the change in **`CHANGELOG.md`** under the `[Unreleased]` header.
* [ ] Check that commits follow the [Mullvad coding guidelines](https://github.com/mullvad/coding-guidelines)

---
This change is [Reviewable](https://reviewable.io/reviews/mullvad/mullvadvpn-app/2350)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,12 +45,12 @@ in other DEs, but we don't regularly test those.
...
-| OpenVPN                       |    ✓    |   ✓   |   ✓   |         |
-| WireGuard                     |    ✓    |   ✓   |   ✓   |    ✓    |
-| OpenVPN over Shadowsocks      |    ✓    |   ✓   |   ✓   |         |
-| Optional local network access |    ✓    |   ✓   |   ✓   |    ✓    |
+|                               | Windows | Linux | macOS | Android | iOS |
+|-------------------------------|:-------:|:-----:|:-----:|:-------:|:---:|
+| OpenVPN                       |    ✓    |   ✓   |   ✓   |         |     |
+| WireGuard                     |    ✓    |   ✓   |   ✓   |    ✓    |  ✓  |
+| OpenVPN over Shadowsocks      |    ✓    |   ✓   |   ✓   |         |     |
+| Optional local network access |    ✓    |   ✓   |   ✓   |    ✓    |  ✓  |
```
Comment (faern) [inline README.md:53]: LAN access is not really "optional" in our iOS app at the moment. LAN is always allowed currently. There are some new possibilities with iOS 14+ that might allow locking it down, but we don't implement that yet.

Could you update this checkbox to have an asterisk and below the table put something like

```
* The local network is always accessible on iOS with the current implementation
```

Watch out for markdown formatting so it does not become a bullet list.

**Review by faern — CHANGES_REQUESTED** [review]
Thanks for this PR. The readme is indeed a bit outdated in this area.

**Comment by sm0rux** [comment]
Så, nu tror jag att jag fixade till det rätt snyggt.

**Comment by faern** [comment]
Thanks for the contribution :)

**Comment by sm0rux** [comment]
My pleasure :)

**Pull request merged**